apps/api-documenter/src/documenters/MarkdownDocumenter.ts

```diff
@@ -37,7 +37,9 @@ import {
   ApiDeclaredItem,
```

Comment:
Could you add a test case to [build-tests/api-documenter-test](https://github.com/microsoft/rushstack/tree/master/build-tests/api-documenter-test)?  It doesn't need to be exhaustive, just something to show how the alias gets rendered.

Reply:
Added test case (I think I did it right. I couldn't find any info about how to actually run it locally, so I couldn't verify.)